bringing group mentoring program current (#5653)
kubernetes/community: mentoring/programs/group-mentoring.md

# Contributor Ladder Growth Programs
  
These programs place an emphasis on learning in groups for active contributors
that would like to be a reviewer or other leadership roles like a Chair. The 

> groups are semi-structured learning environments over the course of three
> months. Want to get `/lgtm` rights? Be a decision maker in an OWNERS file? This

Reviewer: ```suggestion
groups are semi-structured learning environments over the course of a
release. Want to get `/lgtm` rights? Be a decision maker in an OWNERS file? This
```

Suggest we lock this to the release cycle.

## How This Works
Foundation:  
This program is built around our [community membership guidelines] as well as 
our [Chair] and [Tech Lead] governance descriptions.
These docs show how you can grow through the project in different roles and
provides the requirements to get to each level on our ladder. Building trust is

> the key.

Reviewer: ```suggestion
key.
```

## Cohorts
- No more than 4 mentees per 1 mentor; no more than 8 total in the group
  - this scales better than 1:1 and peers can help each other in a community
  setting
- Everyone is on the same journey 

> Member -> Reviewer | Reviewer -> Owner | Member/Reviewer/etc -> Chair

Reviewer: ```suggestion
Member -> Reviewer | Reviewer -> Approver | SIG Member -> Chair
```

> - Three months

Reviewer: ```suggestion
- Three months/one release cycle
```

- Private slack channel

>  Folks want to know you are dependable and understand the domain.

Reviewer: ```suggestion
 For you to progress, the community wants to know you are dependable and understand the domain.
```

 

## Mentor requirements  
- Github Org Member

> - At least the same level as the goal of the group (example: Reviewers for 

Reviewer: ```suggestion
- At least the same level as the goal of the group (example: Reviewer for 
```

Reviewers) 

> - Time committment: one dedicated day of slack triage and at least a 30 minute

Reviewer: ```suggestion
- Time commitment: facilitate biweekly, hour long zoom sessions plus curriculum planning in between sessions (~1-2h/wk total).
```
